api/.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+apiKey.js
```

Comment:
We could maybe use the [`dotenv`](https://www.npmjs.com/package/dotenv) package to turn the apikeys into environment variables when we run our program.